platform-dom/src/main/java/com/softicar/platform/dom/elements/image/viewer/DomImageViewerImageFrame.java

```diff
@@ -0,0 +1,23 @@
+package com.softicar.platform.dom.elements.image.viewer;
+
+import com.softicar.platform.dom.DomCssClasses;
+import com.softicar.platform.dom.elements.DomDiv;
+import com.softicar.platform.dom.elements.image.viewer.tag.DomImageViewerTag;
+
+/**
+ * An invisible element that contains a {@link DomImageViewerImage} child, and
+ * {@link DomImageViewerTag} children.
+ * <p>
+ * Has the exact same size as the {@link DomImageViewerImage} child, which
+ * allows for precise absolute positioning of the contained
+ * {@link DomImageViewerTag} children.
+ *
+ * @author Alexander Schmidt
+ */
+public class DomImageViewerImageFrame extends DomDiv {
```

Comment:
The frame is necessary because we need to be able to exatly position `DomImageViewerTag` elements as overlays above the displayed page. The Holder has a padding to compensate for the box-shadow of the image (or frame); so if we'd just append the Tags to the Holder, a Tag positioned at `left: 0%` would not be aligned to the left side of the displayed image. This way, sane (padding-independent) positioning of Tags would be impossible.
Since `img` elements cannot have children, the easiest way I saw was to wrap the image in an invisible element that can indeed have children. That is, the frame.

Reply:
I see, but maybe we could name it so that its intent is clear, e.g. `DomImageViewerTagsCanvas` or something, or at least add some Javadoc to explain why it's there, so CSS designers will not add paddings or margins to it

Reply:
Renamed to `DomImageViewerTagsCanvas` as suggested. The Javadoc, including a statement of purpoes, was already on top of that class.